### DISCUSSION TITLE
Presentation mode

### DISCUSSION THREAD

**Discussion by antonreshetov** [massCodeIO/massCode#211] [Ideas]
### What if there will be a presentation mode in the app? 

The mechanics are simple, create a series of markdown snippets in the folder, then click on presentation mode. 

### Who will this be useful for? 

I think it's great for classroom use, team meetings, presentations, or simply reviewing notes on your own.

First drafts. 
![Запись экрана 2022-08-09 в 08 52 33](https://user-images.githubusercontent.com/17480987/183655069-b1406390-7d92-4454-adab-3434a622bc11.gif)

**Comment by glennhefley** [comment]
Are you describing something like Python's[ Jupyter notebooks?](https://jupyter.org)?  That would be cool.

**Reply by antonreshetov** [reply]
I don't want to overcomplicate things, it will just be a markdown rendering.
